packages/cli/src/spec/mockPromptSpec.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Result} from '@bubblewrap/core';
+import {MockPromptForStrings} from './mock/MockPromptForStrings';
+
+async function validationFunction(message: string): Promise<Result<string, Error>> {
```

Comment:
Should we test with a validationFunction that returns something other than a string?

Reply:
I don't see how it could change the bottom line.. every case depends on the implementation but with those tests I made sure that the logic of the class is good and functioning (At least I think so)